tests/js/server/dump/dump-setup-common.inc

```diff
@@ -442,7 +465,67 @@ function createSmartGraph() {
   }
   db[edges].save(eDocs);
   db[orphans].save(vDocs);
-};
+}
+
+function createSmartGraphSingleServer() {
+  if (!isEnterprise) {
+    return;
+  }
+  db._useDatabase("UnitTestsDumpSrc");
+
+  const smartGraphName = "UnitTestDumpSmartGraphSingleServer";
+  const edges = "UnitTestDumpSmartEdgesSingleServer";
+  const vertices = "UnitTestDumpSmartVerticesSingleServer";
+  const orphans = "UnitTestDumpSmartOrphansSingleServer";
+  if (sgm._exists(smartGraphName)) {
+    sgm._drop(smartGraphName, true);
+  }
+  db._drop(edges);
+  db._drop(vertices);
+  db._drop(orphans);
+
+  sgm._create(smartGraphName, [sgm._relation(edges, vertices, vertices)],
+    [orphans], {numberOfShards: 5, smartGraphAttribute: "value"});
```

Comment:
This part could be replaced with `createSmartGraphWithoutData()` with appropriate parameters for the names.

Reply:
Fixed.